examples/DocExampleApis/Program.cs

```diff
@@ -122,4 +137,101 @@ public static async Task Example_API_Delete(CacheClient cacheClient)
             throw new Exception($"An error occurred while attempting to delete key 'test-key' from cache 'test-cache': {error.ErrorCode}: {error}");
         }
     }
+
+    public static async Task Example_API_GenerateDisposableToken(IAuthClient authClient)
+    {
+        var scope = new DisposableTokenScope(Permissions: new List<DisposableTokenPermission>
+        {
+            new DisposableToken.CacheItemPermission(
+                CacheRole.ReadWrite,
+                CacheSelector.ByName("cache"),
+                CacheItemSelector.AllCacheItems
+            ),
+            new DisposableToken.CachePermission(
+                CacheRole.ReadOnly,
+                CacheSelector.ByName("topsecret")
+            ),
+            new DisposableToken.TopicPermission(
+                TopicRole.PublishSubscribe,
+                CacheSelector.ByName("cache"),
+                TopicSelector.ByName("example-topic")
+            )
+        });
+        var tokenResponse = await authClient.GenerateDisposableTokenAsync(
+            scope,
+            ExpiresIn.Minutes(5)
+        );

```

Comment:
This is 100% the fault of the example script I threw together, but I think this approach of creating a custom scope with multiple permissions is possibly over-complicated for the API examples. The Javascript example uses the `DisposableTokenScopes` convenience methods to simplify this process and I think we should probably follow that example. So, for example, the selected code would omit the construction of the `scope` variable and use a predefined scope instead:

```
var tokenResponse = await client.GenerateDisposableTokenAsync(
    DisposableTokenScopes.CacheKeyReadWrite("squirrels", "mo"),
    ExpiresIn.Minutes(5)
);
```

The Javascript example shows off 4 of the predefined scopes, and I think it would be good to show multiple of the `DisposableTokenScopes` here too.

Sorry for causing the extra work and for the over-verbose example code. I'm going to either add a second disposable token example program using the predefined scopes or just add an example of each approach in the one program soon so users can study and compare both approaches, but for these docs I think the simplest approach is the best to show.

Reply:
No worries! Easy enough to fix and better to be consistent with the existing examples for sure. I'll use predefined scopes to address the other 3 examples in the JS version as well.